src/exporters/coreml/__main__.py

```diff
@@ -74,8 +76,8 @@ def main():
     )
     parser.add_argument(
         "--feature",
-        choices=list(FeaturesManager.AVAILABLE_FEATURES),
-        default="default",
+        choices=list(FeaturesManager.AVAILABLE_FEATURES_INCLUDING_LEGACY),
```

Comment:
I arbitrarily decided to deprecate the old names but still support them in the CLI (not the API). Happy to remove all support at once, or allow the legacy names to still be used elsewhere.